pkg/build/nodeimage/buildcontext.go

```diff
@@ -125,7 +125,7 @@ func (c *buildContext) buildImage(bits kube.Bits) error {
 		return err
 	}
 
-	// pre-pull images that were not part of the build and write CNI / storage
+	// pre-pull images that were not part of the build and write CNI / Storage
```

Comment:
Storage isn't a proper name here, I don't think it needs to be capitalized.

Reply:
CNI is only capitalized because it is an acronym AND a proper name. In this case storage is neither.